src/Markdig/Renderers/Html/Inlines/AutolinkInlineRenderer.cs

```diff
@@ -25,14 +25,22 @@ public bool AutoRelNoFollow
             }
             set
             {
-                string rel = "nofollow";
-                if (value && !Rel.Contains(rel))
+                const string NoFollow = "nofollow";
+
+                if (value)
                 {
-                    Rel = string.IsNullOrEmpty(Rel) ? rel : Rel + $" {rel}";
+                    if (string.IsNullOrEmpty(Rel))
+                    {
+                        Rel = NoFollow;
+                    }
+                    else if (!Rel!.Contains(NoFollow))
```

Comment:
This `!` isn't needed on Core 3.1+, sadly older runtimes aren't smart enough to realize `string.IsNullOrEmpty` is a null check

Reply:
There's a NRE condition here that we're not handling correctly. We should rewrite this code to correctly handle a null Rel.

Reply:
It does handle it via `string.IsNullOrEmpty`

Reply:
Indeed. Missed the outer logic changes here. 👍